tests/chainer_tests/functions_tests/array_tests/test_scatter_add.py

```diff
@@ -32,6 +32,12 @@ def setUp(self):
         self.gy_data = numpy.random.uniform(
             -1, 1, self.shape).astype(numpy.float32)
         self.b_data = self.b_data.astype(numpy.float32)
+        self.gga_data = numpy.random.uniform(
+            -1, 1, self.a_data.shape).astype(numpy.float32)
+        self.ggb_data = numpy.random.uniform(
+            -1, 1, self.b_data.shape).astype(numpy.float32)
+        self.check_backward_options = {
```

Comment:
* Shouldn't this be `check_double_backward_options`?
* I think you don't need to explicitly specify `dtype` argument because all inputs are `float32`.

Reply:
Removed the redundant dtype and `check_double_backward_options ` since it is only used at one place.